ledger/accountdb_test.go

```diff
@@ -311,14 +311,15 @@ func TestAccountDBRound(t *testing.T) {
 		require.NoError(t, err)
 		expectedOnlineRoundParams = append(expectedOnlineRoundParams, onlineRoundParams)
 
-		updatedAccts, updatesResources, err := accountsNewRound(tx, updatesCnt, resourceUpdatesCnt, nil, ctbsWithDeletes, proto, basics.Round(i))
+		updatedAccts, updatesResources, updatedBoxes, err := accountsNewRound(tx, updatesCnt, resourceUpdatesCnt, nil, ctbsWithDeletes, proto, basics.Round(i))
```

Comment:
@algoidurovic Minor:  As part of renaming from Boxes-to-KV, can you make these renames?
* Rename `updatedBoxes` to `updatedKVs` to match the method being tested.
* Rename PR title to include a _KV_ mention.  
  * I periodically search PR history and having the associated keywords simplifies historical searches.
  * If you'd prefer, I'm good with retaining a Box reference in the PR title too.

Reply:
Additionally, any updates to the PR title ought to be reflected into the PR description.